### DISCUSSION TITLE
About bugg fix

### DISCUSSION THREAD

**Discussion by KiyoEditz** [ImYanXiao/Elaina-MultiDevice#111] [General]
Ini udah di fix belom bugg gak bisa scan nya, saya gak tau banyak tapi saya kira ini dari main JS nya

**Comment by ImYanXiao** [comment]
Sepertinya memang harus pindah ke pairingCode, hanya saja sering rusak sessionnya

**Reply by KiyoEditz** [reply]
Kek nya jangan bang, capek saya boleh balik scan nanti, device saya satu soal nya, emang masalah nya apa sih
